reference/admin.yaml

```diff
@@ -2112,6 +2112,13 @@ components:
           type: string
           nullable: true
           example: '2020-07-20'
+    is_implicit_merge_blocked:
+      type: boolean
+      nullable: true
+      description: >
+        This indicates if the expense can be considered for implicit merge or
+        not
```

Comment:
just check if this looks good in preview @Code0987 sir.

Reply:
Ohh, it's the bot messing up :/ But it looks fine
<img width="632" alt="Screenshot 2022-05-23 at 5 29 09 PM" src="https://user-images.githubusercontent.com/1825861/169814540-5d9aa032-ab24-43bb-8e4c-315e02f9a6a0.png">